app/views/benefits/ReceiveOwnCarMileageBenefitView.scala.html

```diff
@@ -0,0 +1,60 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.headerFooterTemplate.Layout
+@import views.html.templates.helpers.ErrorSummary
+@import views.html.templates.helpers.Button
+@import views.html.templates.helpers.InputRadio
+
+@this(
+        layout: Layout,
+        inputRadio: InputRadio,
+        formWithCsrf: FormWithCSRF,
+        errorSummary: ErrorSummary,
+        button: Button
+)
+
+@(form: Form[Boolean], taxYear: Int, employmentId: String)(implicit user: User[_], messages: Messages, appConfig: AppConfig)
+
+@titleForUse = @{messages(s"benefits.receiveOwnCarMileageBenefit.title.${if(user.isAgent) "agent" else "individual"}")}
+@headingForUse = @{titleForUse}
+@paragraphText = {
+    <p class="govuk-body">@messages(s"benefits.receiveOwnCarMileageBenefit.needToKnow.${if(user.isAgent) "agent" else "individual"}")</p>
+    <p>@messages(s"benefits.receiveOwnCarMileageBenefit.check.${if(user.isAgent) "agent" else "individual"}")</p>
```

Comment:
think the second p tag needs the class="govuk-body" bit as well

Reply:
yes - added